libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/GranteeItem.tsx

```diff
@@ -0,0 +1,112 @@
+// (C) 2021 GoodData Corporation
+import React, { useCallback, useMemo } from "react";
+import { useIntl } from "react-intl";
+import cx from "classnames";
+import {
+    DialogModeType,
+    GranteeItem,
+    IGranteeGroup,
+    IGranteeGroupAll,
+    IGranteeItemProps,
+    IGranteeUser,
+} from "./types";
+import { getGranteeLabel, getGranteeItemTestId } from "./utils";
+import { GranteeGroupIcon, GranteeOwnerRemoveIcon, GranteeRemoveIcon, GranteeUserIcon } from "./GranteeIcons";
+import { Button } from "../../../Button";
+
+interface IGranteeUserItemProps {
+    grantee: IGranteeUser;
+    mode: DialogModeType;
+    onDelete: (grantee: GranteeItem) => void;
+}
+
+interface IGranteeGroupItemProps {
+    grantee: IGranteeGroup | IGranteeGroupAll;
+    mode: DialogModeType;
+    onDelete: (grantee: GranteeItem) => void;
+}
+
+const GranteeUserItem: React.FC<IGranteeUserItemProps> = (props) => {
+    const { grantee, mode, onDelete } = props;
+    const intl = useIntl();
+
+    const granteeLabel = useMemo(() => {
+        const name = getGranteeLabel(grantee, intl);
+        const you = intl.formatMessage({ id: "shareDialog.share.grantee.item.you" });
+        return grantee.isCurrentUser ? `${name} (${you})` : name;
```

Comment:
I would construct this string via localization, i.e., `<FormatMessage id="shareDialog.share.grantee.item.you" params={userName: name} />` and change `shareDialog.share.grantee.item.you` to `"{userName} (You)"` in `en-US.json`.

Reply:
done